src/emails/EmailService.ts

```diff
@@ -0,0 +1,33 @@
+import { User, UserModel } from './../schemas/User';
+
+const sendmail = require('sendmail')();
```

Comment:
The dependency has not been added to the package.json, or at least, it file was not been commited, therefore an error was occurred.